website/docs/r/service_level_objective.html.markdown

```diff
@@ -96,17 +96,16 @@ The following options are specific to the `type` of service level objective:
         * `numerator` - (Required) the sum of all the `good` events
         * `denominator` - (Required) the sum of the `total` events
         * Example Usage:
-        
+
 ```hcl
 query {
     numerator   = "sum:my.custom.count.metric{type:good}.as_count()"
-    denominator = "sum:my.custom.count.metric{*}.as_count()" 
+    denominator = "sum:my.custom.count.metric{*}.as_count()"
 }
 ```
 
 * `monitor` type SLOs:
-    * `monitor_ids` - (Optional) A list of numeric monitor IDs for which to use as SLIs. Their tags will be auto-imported into `monitor_tags` field in the API resource. At least 1 of `monitor_ids` or `monitor_search` must be provided.
-    * `monitor_search` - (Optional) The monitor query search used on the monitor search API to add monitor_ids by searching. Their tags will be auto-imported into `monitor_tags` field in the API resource. At least 1 of `monitor_ids` or `monitor_search` must be provided.
+    * `monitor_ids` - (Optional) A list of numeric monitor IDs for which to use as SLIs. Their tags will be auto-imported into `monitor_tags` field in the API resource.
```

Comment:
If one of `monitor_ids` and `monitor_search` must be provided and we are hiding `monitor_search`, shouldn't we then also make `monitor_ids` required for now?

Reply:
@bkabrda that makes sense to me.  I haven't actually updated any code here...should we just document that `monitor_ids` is required for now?

Reply:
Adjusted so `monitor_ids` is required

Reply:
@bkabrda this should be ready for another review when you can